### ISSUE TITLE
[Feature Request] Refresh data when clicking menu bar item.

### ISSUE THREAD

**Issue by Sangeppato** [exelban/stats#2140]
Hello, I don't know whether this is a bug or a feature request (I suppose the latter). I like to keep a "high" refresh interval (5s) to avoid excessive energy usage, but when I click on the menu bar item to view the details in the pop-up, I think it would make sense for the stats to refresh immediately (i.e., to force an update), as we can assume that in this case the user is actively monitoring the system.
Thank you.

**Comment by Chiroyce1** [comment]
Yeah sometimes when i click the menu bar item the data in the popup is actually not instantly updated (like the graphs). I would love to keep the update frequency 10s or 15s and have it update immediately when clicked.

**Comment by exelban** [comment]
Hi. Yeap, I think there could be some improvements in opening the popup view. But for sure there will be no other update interval for popup view.

**Comment by Sangeppato** [comment]
Hi @exelban, thank you for the response. I was not suggesting a different update "interval" for the popup view, but only a "forced update" whenever the popup is clicked. This way, a user wouldn't have to wait for the next refresh to occur.

**Comment by PuzzledUser** [comment]
I would appreciate this feature too, but care must be taken as the graphs then will have an extra data point out of sequence. I guess this just replaces the sample before or after the click, but this could cause a discontinuity in the graphs.
